.github/workflows/scripts/check-image-vulnerabilities.py

```diff
@@ -7,6 +7,14 @@
 
 QUAY_ORG="rhacs-eng"
 
+VULNERABILITY_LEVELS = {
```

Comment:
Nit: It is actually a list of `VULNERABILITY_SEVERITIES`, but `SEVERITIES` would be also fine here.